social-distancing/libs/Core.py

```diff
@@ -0,0 +1,160 @@
+import os, re
+import time
+
+import numpy as np
+import cv2 as cv
+from scipy.spatial import distance as dist
+from libs.centroid_object_tracker import CentroidTracker
+
+class Distancing:
+
+    def __init__(self, config):
+        self.config = config
+        self.ui = None
+        self.detector = None
+        self.device = self.config.get_section_dict('Detector')['Device']
+        self.running_video = False
+        self.tracker = CentroidTracker(maxDisappeared=5)
+        if self.device == 'Jetson':
+            from libs.detectors.jetson.Detector import Detector
+            self.detector = Detector(self.config)
+        elif self.device == 'EdgeTPU':
+            from libs.detectors.edgetpu.Detector import Detector
+            self.detector = Detector(self.config)
+        elif self.device == 'Dummy':
+            self.detector = None
+
+        self.image_size = [int(i) for i in self.config.get_section_dict('Detector')['ImageSize'].split(',')]
+        
+        if self.device != 'Dummy':
+            print('Device is: ', self.device)
+            print('Detector is: ', self.detector.name)
+            print('image size: ', self.image_size)
+
+    def set_ui(self, ui):
+        self.ui = ui
+
+    def __process(self, cv_image):
+        """
+        return object_list list of  dict for each obj, 
+        obj["bbox"] is normalized coordinations for [x0, y0, x1, y1] of box
+        """
+        if self.device == 'Dummy': 
+            return cv_image, [], None
+
+        resized_image = cv.resize(cv_image, tuple(self.image_size[:2]))
+        rgb_resized_image = cv.cvtColor(resized_image, cv.COLOR_BGR2RGB)
+        tmp_objects_list = self.detector.inference(rgb_resized_image)
+        hscale = cv_image.shape[0]/resized_image.shape[0]
```

Comment:
can you please remove hscale and wscale variables? They don't seem to be used anywhere in this function.